packages/react/src/components/AccountSettings/index.ts

```diff
@@ -0,0 +1 @@
+export * from './ChangePassword';
```

Comment:
Should we be exporting everything from `ChangePassword`?

Reply:
Made more explicit: https://github.com/aws-amplify/amplify-ui/blob/97936eec8e4fce7511c6c3c5deee9aed98b9984b/packages/react/src/components/AccountSettings/index.ts